src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,56 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, V> {
+    private T first;
+    private V second;
 
+    public Pair(T first, V second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public T getFirst() {
+        return first;
+    }
+
+    public V getSecond() {
+        return second;
+    }
+
+    public static <T, V> Pair<T, V> of(T first, V second) {
+        Pair<T, V> pair = new Pair<T, V>(first, second);
```

Comment:
```suggestion
        return new Pair<>(first, second);
```
writing just `<>` here is possible because of the [diamond operator](https://www.baeldung.com/java-diamond-operator)